src/static/authors.tex

```diff
@@ -0,0 +1,487 @@
+
+%FLAG
+\author[0000-0003-4189-3450]{Henrik~Norman}
+\affiliation{Winter Way, Lerum, Sweden}
+
+\author[0000-0002-2432-8946]{Richard~O'Steen}
+\affiliation{\affstsci}
+
+\author[0000-0001-9857-7788]{Kyle~A.~Oman}
+\affiliation{Institute for Computational Cosmology, Durham University, South Road, Durham DH1 3LE, United Kingdom}
+\affiliation{Department of Physics, Durham University, South Road, Durham DH1 3LE, United Kingdom}
+
+\author{Camilla~Pacifici}
+\affiliation{\affstsci}
+
+\author[0000-0002-9351-6051]{Sergio~Pascual}
+\affiliation{Departamento de F\'{i}sica de la Tierra y Astrof\'{i}sica, Fac. CC. F\'{i}sicas, Universidad Complutense de Madrid, E28040, Madrid, Spain}
+\affiliation{Instituto de F\'{i}sica de Part\'{i}culas y del Cosmos, IPARCOS, Fac. CC. F\'{i}sicas, Universidad Complutense de Madrid, E28040, Madrid, Spain}
+
+\author[0000-0003-0139-6951]{J.~Pascual-Granado}
+\affiliation{Instituto de Astrof\'{\i}sica de Andaluc\'{\i}a (CSIC). Glorieta de la Astronom\'{\i}a s/n. 18008, Granada, Spain}
+
+%FLAG
+\author[0000-0002-5923-4966]{Rohit~R~Patil}
+\affiliation{Syracuse University, Syracuse, NY}
+
+\author[0000-0002-1852-9653]{Gabriel~I~Perren}
+\affiliation{Instituto de Astrof\'isica de La Plata (IALP-CONICET), La Plata, Argentina}
+
+%FLAG
+\author[0000-0002-9427-5448]{Timothy~E~Pickering}
+\affiliation{MMT Observatory}
+
+\author{Tanuj~Rastogi}
+\noaffiliation
+
+\author[0000-0002-9453-7735]{Benjamin~R.~Roulston}
+\affiliation{\affcfa}
+\affiliation{Department of Astronomy, Boston University, 725 Commonwealth Avenue, Boston, MA 02215, USA}
+
+\author[0000-0001-8661-3825]{Daniel~F~Ryan}
+\affiliation{University of Applied Sciences and Arts Northwest Switzerland, Bahnhofstrasse 6, 5210 Windisch, Switzerland}
+\affiliation{American University, 4400 Massachusetts Avenue, NW Washington, DC 20016, USA}
+
+\author[0000-0001-9376-3135]{Eli~S.~Rykoff}
+\affiliation{SLAC National Accelerator Laboratory, Menlo Park, CA 94025, USA}
+
+\author[0000-0003-1149-6294]{Jose~Sabater}
+\affiliation{STFC UK Astronomy Technology Centre, Royal Observatory, Blackford Hill, Edinburgh, EH9 3HJ, UK}
+
+%FLAG
+\author[0000-0002-9523-5640]{Parikshit~Sakurikar}
+\affiliation{International Institute of Information Technology - Hyderabad}
+
+%FLAG
+\author[0000-0002-3680-4364]{Jes\'us~Salgado}
+\affiliation{SKAO Observatory, Jodrell Bank, Cheshire, UK}
+
+\author[0000-0002-1838-4757]{Aniket~Sanghi}
+\affiliation{Department of Astronomy, The University of Texas at Austin, 2515 Speedway, C1400, Austin, TX 78712, USA}
+
+\author[0000-0003-2657-3889]{Nicholas~Saunders}
+\affiliation{Institute for Astronomy, University of Hawaiʻi at M\=anoa, 2680 Woodlawn Drive, Honolulu, HI 96822, USA}
+\affiliation{American Museum of Natural History, 200 Central Park West, Manhattan, NY 10024, USA}
+\altaffiliation{NSF Graduate Research Fellow}
+
+%FLAG
+\author[0000-0001-6353-0808]{Volodymyr~Savchenko}
+\affiliation{University of Geneva, Department of Astronomy}
```

Comment:
I saw the PR, wanted to check if my affiliations are right, and realized it's flagged.

Is it ok to provide an update like this?

```suggestion
\affiliation{Department of Astronomy, University of Geneva, Chemin d'\'Ecogia, 16 CH-1290 Versoix, Switzerland}
\affiliation{EPFL SCITAS, Station 9, CH-1015 Lausanne, Switzerland}
```

Reply:
Nice - thanks for being diligent 😄